explorer/explorermiddleware.go

```diff
@@ -48,7 +48,7 @@ func (exp *explorerUI) BlockHashPathOrIndexCtx(next http.Handler) http.Handler {
 			if height > maxHeight {
 				expectedTime := time.Duration(height-maxHeight) * exp.ChainParams.TargetTimePerBlock
 				message := fmt.Sprintf("This block is expected to arrive in approximately in %v. ", expectedTime)
-				exp.StatusPage(w, defaultErrorCode, message, NotFoundStatusType)
+				exp.StatusPage(w, "Future Block", message, NotFoundStatusType)
```

Comment:
please define a new error message alongside the others in https://github.com/decred/dcrdata/blob/0dfef89dbc01b471063d1f4e41a88bc24ae6421e/explorer/explorerroutes.go#L30

Reply:
OK , I created this const : unminedBlock

const (
        defaultErrorCode    = "Something went wrong..."
        defaultErrorMessage = "Try refreshing this page... it usually fixes things"
        fullModeRequired    = "full-functionality mode required for this page"
        unminedBlock = "Future Block"
)
So this issue is fixed